packages/@coorpacademy-player-services/src/types.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import type {Action, Content} from '@coorpacademy/progression-engine';
+
+type Url = string;
+
+type JwpOptions = {
+  customProps: {
+    aspectratio: '16:9' | '4:3',
+    autostart: boolean,
+    skin: {
+      name: string
+    },
+    width: string
+  },
+  file: Url,
+  licenseKey: string,
+  playerId: string,
+  playerScript: Url
+};
+
+type Resource = {
+  ref: string,
+  type: 'video' | 'pdf',
+  mimeType: 'video/mp4' | 'application/vimeo',
```

Comment:
IMO this should be in another type like `ResourceMimeType`

Reply:
✅